Sources/BeeDataset/Visualization.swift

```diff
@@ -1,76 +1,76 @@
-// Copyright 2020 The SwiftFusion Authors. All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+// // Copyright 2020 The SwiftFusion Authors. All Rights Reserved.
```

Comment:
removed because plotly no longer supports this. I think we should delete this file entirely?